src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -294,11 +294,10 @@ public TagHelperContent EndTagHelperWritingScope()
 
                 foreach (var attribute in tagHelperOutput.Attributes)
                 {
-                    var value = HtmlEncoder.HtmlEncode(attribute.Value);
                     writer.Write(' ');
                     writer.Write(attribute.Key);
                     writer.Write("=\"");
-                    writer.Write(value);
+                    WriteTo(writer, HtmlEncoder, attribute.Value, escapeQuotes: true);
```

Comment:
- huh?  that code would need to exist two of the three places that call this method.  better to have one special case than two.